src/conversions/chrono.rs

```diff
@@ -0,0 +1,681 @@
+                        py_datetime, datetime,
+                        "{}: {} != {}",
+                        name, datetime, py_datetime
+                    );
+                })
+            };
+
+        check_fixed_offset("fold", 2014, 5, 6, 7, 8, 9, 1_999_999, 999_999, true);
+        check_fixed_offset("non fold", 2014, 5, 6, 7, 8, 9, 999_999, 999_999, false);
+
+        Python::with_gil(|py| {
+            // extract utc with fixedoffset should fail
+            // but fixedoffset from utc seemed to work, maybe because it is also considered fixedoffset?
+            let py_tz = Utc.to_object(py);
+            let py_tz = py_tz.cast_as(py).unwrap();
+            let py_datetime =
+                PyDateTime::new_with_fold(py, 2014, 5, 6, 7, 8, 9, 999_999, Some(py_tz), false)
+                    .unwrap();
+            assert!(py_datetime.extract::<DateTime<FixedOffset>>().is_ok());
+            let offset = FixedOffset::east(3600);
+            let py_tz = offset.to_object(py);
+            let py_tz = py_tz.cast_as(py).unwrap();
+            let py_datetime =
+                PyDateTime::new_with_fold(py, 2014, 5, 6, 7, 8, 9, 999_999, Some(py_tz), false)
+                    .unwrap();
+            assert!(py_datetime.extract::<DateTime<Utc>>().is_err());
+        })
+    }
+
+    #[test]
+    fn test_pyo3_offset_fixed_topyobject() {
+        Python::with_gil(|py| {
+            let py_module = py.import("datetime").unwrap();
+            let py_timezone = py_module.getattr("timezone").unwrap();
+            let offset = FixedOffset::east(3600).to_object(py);
+            let py_timedelta = PyDelta::new(py, 0, 3600, 0, true).unwrap();
+            let py_timedelta = py_timezone.call1((py_timedelta,)).unwrap();
+            assert!(offset.as_ref(py).eq(py_timedelta).unwrap());
+            let offset = FixedOffset::east(-3600).to_object(py);
+            let py_timedelta = PyDelta::new(py, 0, -3600, 0, true).unwrap();
+            let py_timedelta = py_timezone.call1((py_timedelta,)).unwrap();
+            assert!(offset.as_ref(py).eq(py_timedelta).unwrap());
+        })
+    }
+
+    #[test]
+    fn test_pyo3_offset_fixed_frompyobject() {
+        Python::with_gil(|py| {
+            let py_module = py.import("datetime").unwrap();
+            let py_timezone = py_module.getattr("timezone").unwrap();
+            let py_timedelta = PyDelta::new(py, 0, 3600, 0, true).unwrap();
+            let py_tzinfo = py_timezone.call1((py_timedelta,)).unwrap();
+            let offset: FixedOffset = py_tzinfo.extract().unwrap();
+            assert_eq!(FixedOffset::east(3600), offset);
+        })
+    }
+
+    #[test]
+    fn test_pyo3_offset_utc_topyobject() {
+        Python::with_gil(|py| {
+            let utc = Utc.to_object(py);
+            let py_module = py.import("datetime").unwrap();
+            let py_timezone = py_module.getattr("timezone").unwrap();
+            let py_utc = py_timezone.getattr("utc").unwrap();
```

Comment:
`PyDateTime_TimeZone_UTC` can probably be used, IIRC pyo3 have a wrapper for this.

Reply:
yes, fixed